.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: ruby
 script: bundle exec rake
 rvm:
-  - 2.6.5
+  - 2.7.1
```

Comment:
I think we just want to add the build until we are officially off the old ruby (e.g. VM is updated and we've redeployed app)

Reply:
Restored